querybook/webapp/components/Search/SearchResultItem.scss

```diff
@@ -30,11 +30,23 @@
             .result-items-top {
                 padding-right: var(--padding-sm);
                 align-items: baseline;
+                width: 100%;
 
-                .result-item-title {
+                > div:first-child {
+                    flex-grow: 1;
                     overflow: hidden;
+                    .Text {
+                        width: 100%;
+                        .result-item-title {
+                            overflow-wrap: break-word;
+                        }
+                    }
+                }
+
+                > div:last-child:not(:first-child) {
```

Comment:
for the timestamp, could you add a class name for it, instead of using last-child?

and I think you only need something like, without min-width
```
.result-item-timestamp {
  flex-shrink: 0
}
```

Reply:
<img width="723" alt="Screenshot 2023-05-12 at 1 41 05 PM" src="https://github.com/pinterest/querybook/assets/29313935/61f22b79-3fc0-4a87-b7d5-5aea9c97a153">

Reply:
also without min-width, the length of the title will vary based on the length of the timestamp, which looks messy